CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 - `sandstone/PopupTabLayout` to collapse its tab only when a user enters a menu
 - `sandstone/Scroller` focus rule to match latest UX when `focusableScrollbar` prop is `byEnter`
+- `sandstone/Slider` to show active style when touch dragging
```

Comment:
I think we don't need this changelog.
instead we need changelog for https://github.com/enactjs/enact/pull/2869 and general touch support

Reply:
I updated as we discussed.